tests/test_integration/mock_charm/tests/test_interfaces/test_ingress_interface.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Canonical
+# See LICENSE file for licensing details.
+from unittest.mock import patch
+
+import pytest
+from interface_tester import InterfaceTester
+
+
+def test_ingress_interface(interface_tester: InterfaceTester):
+    interface_tester.configure(
+        repo="https://github.com/PietroPasotti/charm-relation-interfaces",
```

Comment:
Again, please point to the Canonical instance of the repo

Reply:
can't do. See the comment right under that line.
The reason is that the charm-relation-interfaces branch that we fetch the ingress interface tests from is *this branch*.
As soon as this branch is merged into main, I'll open another PR to remove that.

Reply:
worked around it by using the same _MockInterfaceTester trick I used in the local e2e tests.

Reply:
worked around it.